dao.go

```diff
@@ -20,10 +20,10 @@ type Dao struct {
 	collection  *mongo.Collection
 }
 
-type DeviceDao interface {
-	AddDevice(device *DevicePayload, ctx context.Context) (primitive.ObjectID, error)
-	GetDevice(id primitive.ObjectID, ctx context.Context) (*Device, error)
-	GetPaginatedDevices(limit, page int, ctx context.Context) ([]Device, error)
+type DevicesDao interface {
```

Comment:
It's good that you have added the context but I don't see you using it - what is the point?